src/styles/styles.js

```diff
@@ -2215,7 +2215,11 @@ const styles = {
     },
 
     twoFactorAuthCodesButton: {
-        minWidth: 100,
+        minWidth: 140,
+    },
+
+    verifyCodesButton: {
```

Comment:
Can you rename the style here to something more close to what the buttons are for? e.g. `twoFactorAuthCopyCodeButton`